operator/src/main/java/oracle/kubernetes/operator/OperatorEventWatcher.java

```diff
@@ -22,10 +22,10 @@
  * This class handles Domain Event watching. It receives event notifications and sends them into the operator
  * for processing.
  */
-public class DomainEventWatcher extends Watcher<V1Event> {
+public class OperatorEventWatcher extends Watcher<V1Event> {
```

Comment:
Do we need to rename `domain events` to `operator events` in the doc?

Reply:
Operator events consist of "domain events" and "namespace events". The doc use "domain events" for events that specific to individual domains. I kept the doc title as "Domain Events" because the most events are domain-specific and the doc is part of the "Managing domains".